GitVersion.yml

```diff
@@ -1,5 +1,10 @@
 mode: ContinuousDeployment
 next-version: 3.0.0
-branches: {}
+branches:
+  release:
+    regex: ^release?[/-]
```

Comment:
how would this work for branches we do work on such as the `release/v2.20.0` that was a WIP until we were ready to ship it?

Reply:
It would work the same. Any branch starting with "release/" that is used to create a pre-release will follow the "<some release num>-beta<a number>" pattern.

Reply:
what about non-prerelease? that is, we create a branch and do some work on it and then we want to release from it. Will this support creating a v2.20.0 with no beta moniker on a release/v2.20.0 branch?

Reply:
You would add something like this to the branches:
```yaml
preview:
  regex: ^preview?[/-]
  tag: feature # or whatever you want the tag to be
  increment: None
  is-release-branch: true
```